.github/CODEOWNERS

```diff
@@ -59,7 +59,7 @@ exporter/jaegerthrifthttpexporter/                       @open-telemetry/collect
 exporter/kafkaexporter/                                  @open-telemetry/collector-contrib-approvers @pavolloffay @MovieStoreGuy
 exporter/loadbalancingexporter/                          @open-telemetry/collector-contrib-approvers @jpkrohling
 exporter/logicmonitorexporter/                           @open-telemetry/collector-contrib-approvers @bogdandrutu @khyatigandhi6 @avadhut123pisal
-exporter/logzioexporter/                                 @open-telemetry/collector-contrib-approvers @jkowall @Doron-Bargo @yotamloe
+exporter/logzioexporter/                                 @open-telemetry/collector-contrib-approvers @mirii1994 @Doron-Bargo @yotamloe
```

Comment:
Please make this a separate PR. Is @mirii1994 a member of the OpenTelemetry organization?

Reply:
She is not, will send her the instructions, and will split the codeowner changes to a seperate PR.